connectors/sources/servicenow.py

```diff
@@ -324,9 +324,20 @@ async def filter_services(self, configured_service):
                 ]
                 async for table_data in self.get_data(batched_apis=batched_apis):
                     for mapping in table_data:  # pyright: ignore
-                        if mapping["label"] in invalid_services:
-                            servicenow_mapping[mapping["label"]] = mapping["name"]
-                            invalid_services.remove(mapping["label"])
+                        sys_id = mapping.get("sys_id")
```

Comment:
Hopefully this can't be null 🤞 ?

Reply:
It shouldn't be https://www.servicenow.com/community/developer-articles/servicenow-sys-id-importance-of-servicenow-sys-id-servicenow-sys/ta-p/2314412

Reply:
I'm 99.999% sure it can't be, if this is also null then something is very wrong with their table 😅